stdlib/testing/testing.flux

```diff
@@ -5,6 +5,7 @@ import c "csv"
 builtin assertEquals
 builtin assertEmpty
 builtin diff
+builtin fail
```

Comment:
Is this addition absolutely necessary? What's the difference between this and:
```
fail = () => [][0]
```

Reply:
The error message is more clear. I guess this could be used everywhere instead if you're just looking for any error.